src/win32/win32_socket.h

```diff
@@ -9,19 +9,20 @@
 
 #include <io.h>
 #include <winsock2.h>
+
 #if defined(_MSC_VER)
 #pragma comment(lib, "ws2_32.lib")
 #endif
+
 #include <unistd.h>
 
 #if defined(_MSC_VER)
 #pragma warning(pop)
 #endif
 
-/* winsock doesn't feature poll(), so there is a version implemented
- * in terms of select() in mingw.c. The following definitions
- * are copied from linux man pages. A poll() macro is defined to
- * call the version in mingw.c.
+/* winsock doesn't feature poll(), so there is a version implemented in terms of select() in mingw.c.
+ * The following definitions are copied from linux man pages.
+ * A poll() macro is defined to call the version in mingw.c.
```

Comment:
We have changed the file name. Because of it, it should be also changed in the comment.

Reply:
Will be fixed in the next commit.